tests/services/schema_tests.rs

```diff
@@ -0,0 +1,59 @@
+use std::path::PathBuf;
+
+use cedar_agent::schema::load_from_file::load_schema_from_file;
+use cedar_agent::schema::memory::MemorySchemaStore;
+use cedar_agent::policies::memory::MemoryPolicyStore;
+use cedar_agent::SchemaStore;
+use cedar_agent::PolicyStore;
+
+use crate::services::utils;
+
+#[tokio::test]
+async fn memory_tests() {
+    let store = MemorySchemaStore::new();
+
+    let schema = store.get_schema().await;
+    assert!(schema.is_empty());
+    let updated_schema = store.update_schema(utils::schema()).await;
+    assert!(!updated_schema.is_err());
+    assert!(!updated_schema.unwrap().is_empty());
+
+    let error_schema = store.update_schema(utils::parse_error_schema()).await;
+    assert!(error_schema.is_err());
+    store.delete_schema().await;
+    let entities = store.get_schema().await;
+    assert!(schema.is_empty());
+}
+
+#[tokio::test]
+async fn test_load_schema_from_file() {
+    let schema = load_schema_from_file(PathBuf::from("./examples/schema.json"))
+        .await
+        .unwrap();
+    assert!(!schema.is_empty());
+}
+
+#[tokio::test]
+async fn test_validate_policy() {
+    let policy_store = MemoryPolicyStore::new();
+    let schema_store = MemorySchemaStore::new();
+    let _ = schema_store.update_schema(utils::schema()).await.unwrap();
```

Comment:
why have the `let _ =` ?

Reply:
Good question! I probably intended to save it into a variable but then changed my mind, compiler must have complained about unused value and I ended up doing this for some reason rather than just removing the assignment completely. Removed now.